### DISCUSSION TITLE
优化页面创建逻辑

### DISCUSSION THREAD

**Discussion by sysfox** [Qexo/Qexo#565] [Ideas]
Qexo现有的页面创建逻辑是：
当我在Qexo内创建页面，如`music`，Qexo会在`source`文件夹下创建`music.md`，也就是说得到的最终访问链接是:`address/music.html`

然而大部分人喜欢访问或创建`/music/`这么个文件夹，希望Qexo可以在设置里面提供选项，允许用户选择是在`source`文件夹下创建`xxx.md`还是在`source`文件夹下创建`/xxx/index.md`

**Comment by am-abudu** [comment]
不同博客系统对应的新建配置不同，请问你是用的是Hexo还是Hugo

**Reply by am-abudu** [reply]
我正准备优化文件新建方式

**Comment by sysfox** [comment]
Hexo

---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年08月25日 15:57 |
| 收件人 | Qexo/Qexo ***@***.***> |
| 抄送人 | Teror ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [Qexo/Qexo] 优化页面创建逻辑 (Discussion #565) |

我正准备优化文件新建方式
